src/main/java/com/zendesk/maxwell/schema/ddl/MysqlParserListener.java

```diff
@@ -166,7 +166,7 @@ public void exitAdd_column_parens(mysqlParser.Add_column_parensContext ctx) {
 		while ( this.columnDefs.size() > 0 ) {
 			ColumnDef c = this.columnDefs.removeFirst();
 			// unable to choose a position in this form
-			alterStatement().columnMods.add(new AddColumnMod(c.getName(), c, null));
+			alterStatement().columnMods.add(new AddColumnMod(c.getName(), c, new ColumnPosition()));
```

Comment:
did `null` work before? seems to me it'd throw an exception when calling `index` on `null` (AddColumnMod apply method)